### DISCUSSION TITLE
Multi-line facet strings

### DISCUSSION THREAD

**Discussion by captain-yoshi** [observablehq/plot#1054] [Q&A] [answered]
I can add manually newlines with `Plot.text()` but is there a way to achieve that using facets ? I just want to add a title to my plot ! Using the x facet gives the right position for the title, but maybe there are better ways.

```js
Plot.plot({
  height: 600,
  grid: true,
  facet: {
    data: penguins,
    x: "I want a newline \n here",
    y: "species",
    marginRight: 80
  },
  ...
})
```
![image](https://user-images.githubusercontent.com/32679594/191331366-b73da952-b7b7-487a-b362-8b47fbf4c22c.png)

**Comment by Fil** [comment, marked as answer]
That's an unexpected use of faceting :). Plot has no title option yet, see #92.

You could alternatively use:
    Plot.text(["My title"], {frameAnchor: "top", dy: -20, lineAnchor: "bottom", text: d => d})

See https://observablehq.com/@recifs/a-centered-title
